### PR TITLE
Support for East Asian numeric prefix

### PR DESCRIPTION
**Pull request by akihironagai**
Support for East Asian numeric prefix in "Top read".

In East Asian language, numbers are grouped into fours.

万 (萬, 만) = 10^4

1,000 = 1 k = 1000
1,0000 = 10 k = 1 万
1,00000 = 100 k = 10 万
1,000,000 = 1 M = 100 万
1,000,0000 = 10 M = 1000 万

|Before|After|
|---|---|
|Screenshot_20211220-002322_Wikipedia Beta|Screenshot_20211220-002317_Wikipedia Alpha|

### CODE REVIEW

**Comment by dbrant** [comment]
@akihironagai Thanks for the PR.  As it is right now, this doesn't work for us because our string translation files are auto-generated from our crowdsourced translation site (https://translatewiki.net).

Can all this logic be made even simpler by using `icu.text.CompactDecimalFormat` instead?

**Comment by akihironagai** [comment]
> Can all this logic be made even simpler by using icu.text.CompactDecimalFormat instead?

I've implemented the logic using CompactDecimalFormat.
As a result, `view_top_read_card_pageviews_k_suffix` and `view_top_read_card_pageviews_m_suffix` are no longer needed from `strings.xml`.

**Review by cooltey — APPROVED** [review]

File: app/build.gradle [hunk 1]
```diff
@@ -216,6 +216,7 @@ dependencies {
     implementation "com.github.bumptech.glide:okhttp3-integration:$glideVersion"
     kapt("com.github.bumptech.glide:compiler:$glideVersion")
 
+    implementation 'com.ibm.icu:icu4j:70.1'
```
Comment (dbrant) [inline app/build.gradle:219]: We would rather not import a whole new dependency just for this single feature. We can just use `android.icu.text.CompactDecimalFormat`, and make it conditional for API 24+.

**Review by dbrant — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (akihironagai) [reply app/build.gradle]: I've fixed it.

**Pull request merged**